app/src/main/java/cm/aptoide/pt/app/AppViewManager.java

```diff
@@ -441,17 +343,12 @@ public void handleAdsLogic(SearchAdResult searchAdResult) {
     adsManager.handleAdsLogic(searchAdResult);
   }
 
-  private boolean isAppcApp() {
-    return cachedAppCoinsViewModel != null && (cachedAppCoinsViewModel.hasAdvertising()
-        || cachedAppCoinsViewModel.hasBilling());
-  }
-
-  public Completable appBought(String path) {
-    return Completable.fromAction(() -> {
-      cachedApp.getPay()
+  public Completable appBought() {
+    return getAppModel().doOnSuccess(appModel -> {
+      appModel.getPay()
           .setPaid();
-      cachedApp.setPath(path);
```

Comment:
Yeah I removed it at the time, because it seemed like it wasn't doing anything but seems like I was wrong :<... Since we will be removing support it probably won't matter much but I'll add it back.